fixtures/parallelizer/__init__.py

```diff
@@ -251,7 +251,9 @@ def __init__(self, config):
             request = self.sprout_client.request_check(self.sprout_pool)
             self.appliances = []
             # Push an appliance to the stack to have proper reference for test collection
-            IPAppliance(address=request["appliances"][0]["ip_address"]).push()
+            # FIXME: this is a bad hack based on the need for controll of collection partitioning
```

Comment:
actually its fine if we pull in a appliance after item collection right for modify-items

we should consider making accessing them more inexpensive in some ways